docs/support-bundle-guide.md

```diff
@@ -0,0 +1,253 @@
+
+```bash
+$ kubectl get supportbundlecollections support-bundle-for-nodes -ojson
+
+...
+ "status": {
+        "collectedNodes": 1,
+        "conditions": [
+            {
+                "lastTransitionTime": "2022-12-08T06:49:35Z",
+                "status": "True",
+                "type": "Started"
+            },
+            {
+                "lastTransitionTime": "2022-12-08T06:49:41Z",
+                "status": "True",
+                "type": "BundleCollected"
+            },
+            {
+                "lastTransitionTime": "2022-12-08T06:49:35Z",
+                "status": "False",
+                "type": "CollectionFailure"
+            },
+            {
+                "lastTransitionTime": "2022-12-08T06:49:41Z",
+                "status": "True",
+                "type": "Completed"
+            }
+        ],
+        "desiredNodes": 1
+    }
+```
+
+The collection should include three tarballs. To access these files, you can
+download the files from the SFTP server `yourtestdomain.com`. There will be two
+tarballs for `support-bundle-for-nodes` including
+"support-bundle-for-nodes_worker1.tar.gz" and "support-bundle-for-nodes_worker2.tar.gz",
+and two for `support-bundle-for-vms` including "support-bundle-for-vms_vm1.tar.gz"
+and "support-bundle-for-vms_vm2.tar.gz" in the `/root/test` folder. Run the
+`tar xvf $TARBALL_NAME` command to extract the files from the tarballs.
+
+## List of collected items
+
+Depending on the methods you use to collect the support bundle, the contents in
+the bundle may differ. The following table shows the differences.
+
+We use `agent`,`controller` and `outside` to represent running command
+`antctl supportbundle` in Antrea Agent, Antrea Controller and out-of-cluster
+respectively. Also, we use `Node` and `ExternalNode` to represent
+"create SupportBundleCollection CR for Nodes" and "create SupportBundleCollection
+CR for external Nodes".
+
+| Collected Item              | Supported Collecting Method                     | Explanation                                                                                                                                                                                                                                                           |
+|-----------------------------|-------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Antrea Agent Log            | agent, outside, Node, ExternalNode              | Antrea Agent log files                                                                                                                                                                                                                                                |
```

Comment:
It may look better to wrap agent, etc. with ``: `agent`, `outside`, ...

Reply:
updated.